eng/pipelines/libraries/base-job.yml

```diff
@@ -96,7 +96,7 @@ jobs:
 
         - ${{ if ne(parameters.liveRuntimeBuildConfig, '') }}:
           - _runtimeDownloadPath: '$(Build.SourcesDirectory)/artifacts/transport/${{ parameters.runtimeFlavor }}'
-          - _runtimeArtifactName: '$(runtimeFlavorName)Product_${{ parameters.osGroup }}${{ parameters.osSubgroup }}_${{ parameters.archType }}_${{ parameters.liveRuntimeBuildConfig }}'
+          - _runtimeArtifactName: 'CoreLib_${{ parameters.osGroup }}${{ parameters.osSubgroup }}_${{ parameters.archType }}_${{ parameters.liveRuntimeBuildConfig }}'
```

Comment:
This will be braking for theor libraries/run-test-job.yml and also it is breaking the builds of libraries that build against mono.

I believe you will need to introduce an intermediate variable and define it in libraries/build-job, libraries/build-test-job and libraries/run-test-job.yml accordingly.

Reply:
fixed by checking for testScope